proof_test.go

```diff
@@ -637,8 +637,8 @@ func TestStatelessDeserializeMissginChildNode(t *testing.T) {
 		t.Fatal("differing commitment for child #0")
 	}
 
-	if droot.(*InternalNode).children[64] != UnknownNode(struct{}{}) {
-		t.Fatalf("non-nil child #64: %v", droot.(*InternalNode).children[64])
```

Comment:
The point of this test is to ensure that the node is missing, but that it is known from the proof that it is missing.